packages/mongo/collection.js

```diff
@@ -388,8 +389,7 @@ Mongo.Collection._rewriteSelector = function (selector) {
       // override the ones set on $regex.
       if (value.$options !== undefined)
         ret[key].$options = value.$options;
-    }
-    else if (_.contains(['$or','$and','$nor'], key)) {
+    } else if (_.contains(['$or','$and','$nor'], key)) {
```

Comment:
I'm not sure what the temptation would be and I don't personally find that formatting appealing, if that's your own temptation. More specifically: `O(1)` lookup-time doesn't seem applicable here and building that `Object`-dictionary would presumably be a tad slower than the `Array`. 

In similar spirit to the other Underscore replacement this issue addresses, my preference would be: 

```js
['$or','$and','$nor'].includes(key);
```

...but that's outside the scope of this PR and I'd vote against changing it here.  (The curly-bracket readjustment seems welcome though, as the previous formatting seems broken compared to the `else` aside it.